packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -49,7 +49,7 @@ export const DefaultCell: FC<TableCellProps> = props => {
           {value}
         </a>
       )}
-      {showFilters && cell.value && <FilterActions {...props} />}
+      {showFilters && cell.value !== undefined && <FilterActions {...props} />}
```

Comment:
nit: I read somewhere about React best practices that we should try to use ternaries instead.
```suggestion
      {showFilters && cell.value ?  <FilterActions {...props} /> : null}
```

Reply:
It will display 'false' as text if the condition is false, should be:

```jsx
{showFilters && cell.value !== undefined ? <FilterActions {...props} /> : null}
```